ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/fonts/PaywallTheme.kt

```diff
@@ -0,0 +1,56 @@
+package com.revenuecat.purchases.ui.revenuecatui.fonts
+
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.text.TextStyle
+
+@Composable
+internal fun PaywallTheme(
+    fontProvider: FontProvider?,
+    content: @Composable () -> Unit,
+) {
+    if (fontProvider == null) {
+        content()
+    } else {
+        val oldTypography = MaterialTheme.typography
+        val typography = oldTypography.copy(
```

Comment:
Nit: maybe extract this to be an extension method on `Typography` that takes a `fontProvider`?